server.js

```diff
@@ -41,34 +36,50 @@ server.method(require('./lib/markdown').methods);
 server.method(require('./lib/community').methods);
 server.method(require('./lib/changelog').methods);
 
-server.route(require('./lib/routes').routes);
-
-var plugins = [];
-plugins.push({
-    register: require('good'),
-    options: {
-        reporters: [{
-            reporter: require('good-console'),
-            events: {
-                log: '*',
-                response: '*',
-                ops: '*'
+const plugins = [
+    {
+        register: require('good'),
+        options: {
+            ops: {
+                interval: 30 * 1000
```

Comment:
Not sure what interval is reasonable for this, I don't know how logs are managed/rotated on the server. Took a punt on 30s.